### ISSUE TITLE
Is it possible to rename processes?

### ISSUE THREAD

**Issue by Crissium** [bmwant/hapless#40]
When I am launching a process, sometimes I forget to set a recognisable name, and it might be very costly to start a new one with the name parameter set. So, is it possible to add a new 'rename' command?

**Comment by bmwant** [comment]
@Crissium this will be added soon, will let you know once released.

**Comment by bmwant** [comment]
@Crissium released as a new `0.6.0` version

![Screenshot 2024-11-29 at 9 01 28 PM](https://github.com/user-attachments/assets/0be76a5e-6e44-407f-8d16-df51d6fdb5e8)

You can already use it
```bash
pip install --upgrade hapless

hap rename [hap-alias] [new-hap-name]
# e.g. you can invoke with hap ID like
hap rename 4 hap-new-name
# or by hap current name like
hap rename hap-name hap-new-name
```

**Comment by Crissium** [comment]
Thank you!
